scripts/cpplint.py

```diff
@@ -415,7 +415,10 @@
     ])
 
 _C_SYSTEM_DIRECTORIES = frozenset([
+    'SystemConfiguration',
```

Comment:
Thanks for doing this.

If you have any ideas for making this check less prone to manual tweaking, I'd be happy to have them :-).